src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,24 @@
 package core.basesyntax;
 
+import core.basesyntax.model.Figure;
+import core.basesyntax.supplier.FigureSupplier;
+import java.util.Random;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    private static FigureSupplier figureSupplier = new FigureSupplier();
+
+    public static void main(String[] args) {
```

Comment:
Let’s read the task again. "The first half of figures in this array should be generated with random parameters. The other half of the figures should have the same, default parameters." It's not realized in your solution